src/pages/Kusujeong/Login/LoginKsj.js

```diff
@@ -1,6 +1,88 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
 import './LoginKsj.scss';
 
 export default function LoginKsj() {
-  return <div>Login_ksj</div>;
+  const [value, setValue] = useState({
+    id: '',
+    pw: '',
+  });
+  // const [backData, setBackData] = useState({});
+
+  const saveUserValue = e => {
+    const { name, value: newValue } = e.target;
+    const targetValues = { ...value, [name]: newValue };
+    setValue(targetValues);
+  };
+  const isValidate =
+    value.id.includes('@') && value.pw.length > 5 ? true : false;
```

Comment:
조건이 truthy하면 true, 아니면 false값을 할당하는데, 아예 조건 자체를 할당해버리면 됨!
```suggestion
  const isValidate = value.id.includes('@') && value.pw.length > 5;
```